### ISSUE TITLE
Renaming a table while adding a new field/column fails when updating the migration in the database.

### ISSUE THREAD

**Issue by YandyZaldivar** [PomeloFoundation/Pomelo.EntityFrameworkCore.MySql#37]
### Steps to reproduce
Add a new migration renaming a table at the same time of adding a new field/column. Then try to update the database with the migration.

### The issue
Migration database update fails with the following exception:

```
Exception message:

System.ArgumentNullException: Value cannot be null.
Parameter name: property

Stack trace:

   at Microsoft.EntityFrameworkCore.Utilities.Check.NotNull[T](T value, String parameterName)
   at Microsoft.EntityFrameworkCore.Storage.RelationalTypeMapper.FindMapping(IProperty property)
   at Microsoft.EntityFrameworkCore.Migrations.MySqlMigrationsSqlGenerationHelper.ColumnDefinition(String schema, String table, String name, Type clrType, String type, Nullable`1 unicode, Nullable`1 maxLength, Boolean rowVersion, Boolean nullable, Object defaultValue, String defaultValueSql, String computedColumnSql, IAnnotatable annotatable, IModel model, MigrationCommandListBuilder builder)
   at Microsoft.EntityFrameworkCore.Migrations.MigrationsSqlGenerator.ColumnDefinition(AddColumnOperation operation, IModel model, MigrationCommandListBuilder builder)
   at Microsoft.EntityFrameworkCore.Migrations.MigrationsSqlGenerator.Generate(AddColumnOperation operation, IModel model, MigrationCommandListBuilder builder)
   at Microsoft.EntityFrameworkCore.Migrations.MigrationsSqlGenerator.Generate(MigrationOperation operation, IModel model, MigrationCommandListBuilder builder)
   at Microsoft.EntityFrameworkCore.Migrations.MigrationsSqlGenerator.Generate(IReadOnlyList`1 operations, IModel model)
   at Microsoft.EntityFrameworkCore.Migrations.Internal.Migrator.GenerateUpSql(Migration migration)
   at Microsoft.EntityFrameworkCore.Migrations.Internal.Migrator.<MigrateAsync>d__12.MoveNext()
--- End of stack trace from previous location where exception was thrown ---

```

### Further technical details

MySQL version: 5.7.13
Operating system: Win10
Pomelo.EntityFrameworkCore.MySql version: 1.0.0

**Comment by Kagamine** [comment]
Hi @YandyZaldivar, I need more info about this. Can you post your migration csharp file which caused this issue?

**Comment by YandyZaldivar** [comment]
Hi,

Just create a new migration from this definition:

    public class Player
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public int AccountId { get; set; }

        public int Score { get; set; }
    }

to this one:

    [Table("NewName")]
    public class Player
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public int AccountId { get; set; }

        public int Score { get; set; }

        public int NewColumn { get; set; }
    }

It will fail when updating the database with the migration.

Thanks,
Yandy

**Comment by Kagamine** [comment]
Confirmed a bug, we will fix this in 1.0.1 milestone.

**Comment by Kagamine** [comment]
@YandyZaldivar Thank you for reporting this issue, we've fixed this bug in 01f8d70. You can try the latest mysql provider for ef core in myget feed: 
https://www.myget.org/Package/Details/pomelo?packageType=nuget&packageId=Pomelo.EntityFrameworkCore.MySql
